container/openejb-jpa-integration/src/main/java/org/apache/openejb/jpa/integration/eclipselink/OpenEJBServerPlatform.java

```diff
@@ -63,4 +65,11 @@ protected void registerSynchronization_impl(AbstractSynchronizationListener list
             transaction.registerInterposedSynchronization(synchronization);
         }
     }
+
+        @Override
+        public void prepareServerSpecificServicesMBean() {
+            if (getDatabaseSession() != null && shouldRegisterRuntimeBean) {
+                 this.setRuntimeServicesMBean(new MBeanOpenEJBRuntimeServices(getDatabaseSession()));
```

Comment:
ensure it uses mBeanServer which has a toggle "skip jmx" on the instance

Reply:
Good idea, I added a check there